.travis.yml

```diff
@@ -1,39 +1,48 @@
 
 language: cpp
 
-install:
-  - sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
-  - sudo apt-get -qq update
-  - sudo apt-get install -y g++-6 libstdc++-6-dev
-  - sudo apt-get install -y valgrind
-  - sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-6 90
-  - sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-6 90
-
-  # Install bison 3.0.4.
-  - wget http://ftp.gnu.org/gnu/bison/bison-3.0.4.tar.gz
-  - tar -xvzf bison-3.0.4.tar.gz
-  - cd bison-3.0.4
-  - ./configure && make && sudo make install
-  - cd ..
-
-  # Install flex 2.6.4
-  - wget https://github.com/westes/flex/releases/download/v2.6.4/flex-2.6.4.tar.gz
-  - tar -xvzf flex-2.6.4.tar.gz
-  - cd flex-2.6.4
-  - ./configure && make && sudo make install
-  - cd ..
-
-  # Show installed versions.
-  - which g++
-  - g++ -v
-  - bison --version
-  - flex --version
```

Comment:
Can we still print these versions? Just to make it simpler to track which bison/flex etc. versions are provided by Ubuntu 16/20/..